src/OpenLoco/src/World/Industry.cpp

```diff
@@ -535,16 +536,17 @@ namespace OpenLoco
         World::SurfaceElement* surface = tile.surface();
         surface->setIsIndustrialFlag(true);
         surface->setIndustry(industryId);
-        surface->setUpdateTimer((var_EA & 0xE0) >> 5);
-        surface->setGrowthStage((var_EA & 0x7));
+        surface->setUpdateTimer(updateTimer & 0xE0);
```

Comment:
should be &0x7 but tbh you could also just get rid of the &0x7 now as it will do that in `setUpdateTimer`